rules/linux/persistence_user_password_change.toml

```diff
@@ -0,0 +1,89 @@
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule monitors for Linux user account password changes. Threat actors may attempt to change the password of a user
+account to maintain access to a system.
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.file*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Linux User Account Password Change"
+risk_score = 21
+rule_id = "cdf1a39b-1ca5-4e2a-9739-17fc4d026029"
+setup = """## Setup
+
+This rule requires data coming in from Elastic Defend.
+
+### Elastic Defend Integration Setup
+Elastic Defend is integrated into the Elastic Agent using Fleet. Upon configuration, the integration allows
+the Elastic Agent to monitor events on your host and send data to the Elastic Security app.
+
+#### Prerequisite Requirements:
+- Fleet is required for Elastic Defend.
+- To configure Fleet Server refer to the [documentation](https://www.elastic.co/guide/en/fleet/current/fleet-server.html).
+
+#### The following steps should be executed in order to add the Elastic Defend integration on a Linux System:
+- Go to the Kibana home page and click Add integrations.
+- In the query bar, search for Elastic Defend and select the integration to see more details about it.
+- Click Add Elastic Defend.
+- Configure the integration name and optionally add a description.
+- Select the type of environment you want to protect, either Traditional Endpoints or Cloud Workloads.
+- Select a configuration preset. Each preset comes with different default settings for Elastic Agent, you can further customize these later by configuring the Elastic Defend integration policy. [Helper guide](https://www.elastic.co/guide/en/security/current/configure-endpoint-integration-policy.html).
+- We suggest to select "Complete EDR (Endpoint Detection and Response)" as a configuration setting, that provides "All events; all preventions"
+- Enter a name for the agent policy in New agent policy name. If other agent policies already exist, you can click the Existing hosts tab and select an existing policy instead.
+For more details on Elastic Agent configuration settings, refer to the [helper guide](https://www.elastic.co/guide/en/fleet/8.10/agent-policy.html).
+- Click Save and Continue.
+- To complete the integration, select Add Elastic Agent to your hosts and continue to the next section to install the Elastic Agent on your hosts.
+For more details on Elastic Defend refer to the [helper guide](https://www.elastic.co/guide/en/security/current/install-endpoint.html).
+"""
+severity = "low"
+tags = [
+    "Domain: Endpoint",
+    "OS: Linux",
+    "Use Case: Threat Detection",
+    "Tactic: Persistence",
+    "Tactic: Privilege Escalation",
+    "Data Source: Elastic Defend"
+]
+timestamp_override = "event.ingested"
+type = "eql"
+query = '''
+file where host.os.type == "linux" and event.type == "change" and event.action == "rename" and
+file.path == "/etc/shadow" and file.Ext.original.path in ("/etc/nshadow", "/etc/shadow+") and
```

Comment:
Do you need to check the `file.path` and the `file.Ext.original.path`?

Reply:
Yes, because file.Ext.orginal.path is a the path of the temporary file, used to write to by the binary changing the user password. It is then added to the `/etc/shadow` file. When not looking at the original path, it will FP and be noisy as many other modification events exist for the `/etc/shadow` file.

Reply:
Should we include backups and `gshadow` as well? What about potential false-positives with backups such as `/etc/shadow-` or `/etc/nshadow-`?

Reply:
Adding gshadow to the query would be too noisy, and I broadend the scope to just look for existing file.Ext.original.path values to ensure all temporary shadow files are included.